Logging/.CSProject/Logger/Logger.cs

```diff
@@ -8,9 +8,10 @@ namespace Anvil.CSharp.Logging
     /// A context specific instance that provides a mechanism to emit logs through <see cref="Log"/>.
     /// Automatically provides contextual information to <see cref="Log"/> about caller context including:
     ///  - Optional, per instance, message prefix
-    ///  - Caller type name
+    ///  - Logger's owner's type name
+    ///  - Caller method name
     ///  - Caller file path
-    ///  - Caller name
+    ///  - Caller file name
```

Comment:
File name isn't provided by Logger. Just the path.

Caller type name is though